ha/core/system_health/const.py

```diff
@@ -92,4 +92,9 @@ class HEALTH_EVENT_ACTIONS(Enum):
     IGNORE = "ignore"
     UPDATE = "update"
     UPDATEPUBLISH = "updatepublish"
-    PUBLISH = "publish"
\ No newline at end of file
+    PUBLISH = "publish"
+
+# Specific_info attributes
+class SPECIFIC_INFO_ATTRIBUTES(Enum):
+    GENERATION_ID = 'generation_id'
+    FUNCTIONAL_TYPE = 'functional_type'
```

Comment:
We need this during ha_start.
Can you move this to ha/const.py ?

Reply:
These are constants related to system health keys. So I think its okay to keep them here.

Reply:
ok